Data/RectRegion.cs

```diff
@@ -104,13 +111,13 @@ public double Height
             get { return _height; }
             set
             {
-                _height = value;
+                _height = Math.Round(value, DecimalPlaces);
                 if (OwnedTexture != null)
                 {
                     if ((_height + Y) > OwnedTexture.ImageHeight)
                         _height = OwnedTexture.ImageHeight - Y;
                 }
-                if (_height < 0)
+                if (_height < 0.1)
```

Comment:
One minor comment, right now `Subpixel` means `DecimalPlaces` is only 0 or 1.  However, if someone was looking only at this code, they may be confused why this value is hardcoded to 0.1.

We may be able to resolve this with something like:

```
static double min_height = Math.Pow(10.0, (double)-DecimalPlaces);
if (_height < min_height)
  _height = min_height;
```

(same for width)

Reply:
i should have used `if (_height =< 0)`, but your idea is better.
is changed.